extensions/wikia/UserProfilePageV3/UserIdentityBox.class.php

```diff
@@ -20,6 +20,8 @@ class UserIdentityBox {
 	const USER_OCCUPATION_CHAR_LIMIT = 200;
 	const USER_GENDER_CHAR_LIMIT = 200;
 
+	const CACHE_TTL = 60 * 60; // 1 hour
```

Comment:
Cache profile pages for an hour.

This means if a user updates their location in the discussion app it won't be reflected on their profile page for an hour, which isn't super ideal, but far less noticeable than avatars since location is only used on profile pages. This used to be cached indefinitely, so I'm concerned about dropping this TTL too much until we see if there's much of a performance cost.